nixos/modules/services/networking/wpa_supplicant.nix

```diff
@@ -40,7 +40,8 @@ in {
         default = [];
         example = [ "wlan0" "wlan1" ];
         description = ''
-          The interfaces <command>wpa_supplicant</command> will use.
+          The interfaces <command>wpa_supplicant</command> will use. If empty, it will
+          automatically use all wireless interfaces.
```

Comment:
Can you document that this isn't a good idea here?

Reply:
Done.